ui/admin-portal/src/app/services/auth.service.ts

```diff
@@ -149,11 +149,30 @@ export class AuthService {
   }
 
   isCandidateOurs(candidate: ShortCandidate): boolean {
-    return true; //todo
+    let ours = false;
+    const loggedInUser = this.getLoggedInUser()
+    //Must be logged in
+    if (loggedInUser) {
+      ours = candidate.user?.partner?.id === loggedInUser.partner.id;
+    }
+    return ours;
   }
 
   isJobOurs(job: ShortJob): boolean {
-    return true; //todo
+
+    //For now all jobs belong to just the default partner.
+    return this.isDefaultDestinationPartner();
+
+    //todo Eventually when we have proper recruiter partner support, the code will look like this:
```

Comment:
I can definitely see the benefit of putting this TODO here. I personally lean towards having an issue on the board under the appropriate epic (e.g. an epic for recruiter partner support) for this sort of thing. Mostly so it doesn't get missed or forgotten. Let's discuss in team refinement what works best going forward.

Reply:
Definitely agree. I have in fact added an item in an existing issue which refers directly to this TODO.

See #91